### ISSUE TITLE
[WAIT] Make course FAIR, as voted for

### ISSUE THREAD

**Issue by richelbilderbeek** [UPPMAX/programming_formalisms#48]
Currently, we try to do our best to make the course follow the highest standards.

One thing we don't do yet is make our training materials FAIR following the guidelines at [Garcia et al., 2020].

I volunteer to do so, following the paper and the material at https://elixir-europe-training.github.io/ELIXIR-TrP-FAIR-Material-By-Design/#lesson-overview . 

## References

- [Garcia et al., 2020] Leyla Garcia, Bérénice Batut, Melissa L. Burke, Mateusz Kuzak, Fotis Psomopoulos, Ricardo Arcila, Teresa K. Attwood, Niall Beard, Denise Carvalho-Silva, Alexandros C. Dimopoulos, Victoria Dominguez Del Angel, Michel Dumontier, Kim T. Gurwitz, Roland Krause, Peter McQuilton, Loredana Le Pera, Sarah L. Morgan, Päivi Rauste, Allegra Via, Pascal Kahlem, Gabriella Rustici, Celia W.G. Van Gelder, and Patricia M. Palagi. Ten simple rules for making training materials FAIR. PLoS Computational Biology, 16(5):1–9, 2020. [doi:10.1371/journal.pcbi.1007854](https://doi.org/10.1371/journal.pcbi.1007854).

**Comment by richelbilderbeek** [comment]
I vote: yes

Assigning the others to cast their vote too.

**Comment by LarsEklund** [comment]
This is in line with what I want and strive for the course and I vote yes on this, we should implement this as part of the fall course

**Comment by bclaremar** [comment]
vote: yes

**Comment by richelbilderbeek** [comment]
Great, I'll do so :+1:

**Comment by richelbilderbeek** [comment]
Added DOI :+1:

**Comment by bclaremar** [comment]
So nice!

**Comment by richelbilderbeek** [comment]
Waiting on:

- [https://github.com/elixir-europe-training/ELIXIR-TrP-FAIR-training-handbook/issues/89](https://github.com/elixir-europe-training/ELIXIR-TrP-FAIR-training-handbook/issues/890
- [https://github.com/elixir-europe-training/ELIXIR-TrP-FAIR-training-handbook/pull/90](https://github.com/elixir-europe-training/ELIXIR-TrP-FAIR-training-handbook/pull/90)

**Comment by richelbilderbeek** [comment]
For 'Make your training materials usable for trainees' I need to create a page with all learning outcomes. I'll make a separate issue for that :+1:

**Comment by richelbilderbeek** [comment]
I feel this is done. I will re-open when I start doubting this :+1:
